<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogikSim</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/LogikSim.ico</normaloff>:/LogikSim.ico</iconset>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tab_widget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>867</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="action_about_qt"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
   <addaction name="menu_view"/>
   <addaction name="menu_help"/>
  </widget>
  <widget class="QStatusBar" name="status_bar"/>
  <widget class="QToolBar" name="tool_bar">
   <property name="windowTitle">
    <string>Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="history_dock_widget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>176</width>
     <height>300</height>
    </size>
   </property>
   <property name="floating">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>History</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="history_widget">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="ActionStackView" name="action_stack_view"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="component_library_dock_widget">
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>200</height>
    </size>
   </property>
   <property name="floating">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>Component Library</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="component_library_widget">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="LibraryView" name="graphicsView"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_about_qt">
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActionStackView</class>
   <extends>QListView</extends>
   <header>actions/action_stack_view/</header>
  </customwidget>
  <customwidget>
   <class>LibraryView</class>
   <extends>QGraphicsView</extends>
   <header>schematics/library_view/</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
